classes/article/ArticleGalleyDAO.inc.php

```diff
@@ -493,15 +493,15 @@ function getByPubIdType($pubIdType, $contextId = null, $title = null, $author =
 			FROM	submission_galleys g
 				' . ($contextId != null?' JOIN submissions s ON (s.submission_id = g.submission_id)':'') . '
 				LEFT JOIN published_submissions ps ON (ps.submission_id = g.submission_id)
-				LEFT JOIN issues i ON (ps.issue_id = i.issue_id AND i.date_published IS NOT NULL)
+				LEFT JOIN issues i ON (ps.issue_id = i.issue_id)
```

Comment:
Looking at `i.published` to check for a published issue is fine; `issues` entries can either be published (`published=1`) or unpublished (`published=0`). But watch out for the order of the parameter array -- I think moving `getCompletionConditions` up potentially puts them out of order.